packages/utils/src/fetch/handlers.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryParam } from '@lubycon/utils';
+import { Headers } from 'cross-fetch';
+import { convertHeadersToObject } from './utils';
+
+export interface LubyconResponse<T> {
+  headers: Record<string, string>;
+  data: T | null;
+  status: number;
+  statusText: string;
+}
+export async function responseHandler<T>(response: Response): Promise<LubyconResponse<T>> {
+  const headers = convertHeadersToObject(response.headers);
+  const defaultResponse = {
+    headers,
+    status: response.status,
+    statusText: response.statusText,
+  };
+  try {
+    const data = await response.json();
+    return Object.assign(defaultResponse, { data });
```

Comment:
요기는 혹시 `Object.assign`을 사용하신 이유가 있으신가유? 구조 분해 할당 사용하면 더 깔끔할 것 같아서유...!

```suggestion
    return { ...defaultResponse, data };
```